build.include

```diff
@@ -0,0 +1,72 @@
+#!/bin/sh
```

Comment:
why you need this file in a root folder?

Reply:
sorry, included by accident (used for testing)